src/parser2.rs

```diff
@@ -0,0 +1,330 @@
+use pest::prelude::*;
+
+
+impl_rdp! {
+    grammar! {
+        // basic blocks of the language
+        op_or        = { ["or"] }
+        op_wrong_or  = { ["||"] }
+        op_and       = { ["and"] }
+        op_wrong_and = { ["&&"] }
+        op_lte       = { ["<="] }
+        op_gte       = { [">="] }
+        op_lt        = { ["<"] }
+        op_gt        = { [">"] }
+        op_eq        = { ["=="] }
+        op_ineq      = { ["!="] }
+        op_plus      = { ["+"] }
+        op_minus     = { ["-"] }
+        op_times     = { ["*"] }
+        op_slash     = { ["/"] }
+        op_true      = { ["true"] }
+        op_false     = { ["false"] }
+        boolean      = _{ op_true | op_false }
+
+        int   = @{ ["-"]? ~ (["0"] | ['1'..'9'] ~ ['0'..'9']*) }
+        float = @{
+            ["-"]? ~
+                ["0"] ~ ["."] ~ ['0'..'9']+ |
+                ['1'..'9'] ~ ['0'..'9']* ~ ["."] ~ ['0'..'9']+
+        }
+
+        identifier = @{
+            (['a'..'z'] | ['A'..'Z'] | ["_"]) ~
+            (['a'..'z'] | ['A'..'Z'] | ["_"] | ["."] | ['0'..'9'])*
+        }
+        // matches anything between 2 double quotes
+        string = @{ ["\""] ~ (!(["\""]) ~ any )* ~ ["\""]}
```

Comment:
I'm using rule `{ ["\""] ~ (!["\""] ~ (["\\\""] | any))* ~ ["\""] }` for escaped `"` but it doesn't work. @dragostis any ideas?